src/pages/About/components/FeatureCard.tsx

```diff
@@ -0,0 +1,40 @@
+import { Card } from '@/components/UI/Card/Card';
+import { Typography } from '@/components/UI/Typography/Typography';
+import { featureCardStyles, iconStyle, paragraphStyle, titleStyle } from './styles';
+import { IconArrowGuide, IconBracketsAngle, IconFile, IconTarget } from '@tabler/icons-react';
+import { theme } from '@/theme';
+
+export const FeatureCard = (
+    {icon, title, description}:
+    {icon: string, title: string, description: string}
+) => {
+    return (
+        <Card style={featureCardStyles}>
+            {getIcon(icon)}
```

Comment:
Never call a function inside jsx because the object of the function will get recreated on every render
Instead you can make this function a component 

Also please separate this component into another file